modal.js

```diff
@@ -0,0 +1,160 @@
+const fourcards = document.querySelector('.fourcards');
+const popcard = document.querySelector('.deskpopcard');
+
+const projects = [
+  {
+    id: 1,
+    Title: 'Tonic',
+    imglink: './image/profilephoto.png',
+    company: 'CANOPY',
+    developer: 'Back End Dev',
+    year: '2015',
+    description:
+      'A daily selection of privately personalized reads; no accountsor sign-ups required. ',
+    tag1: 'Html',
+    tag2: 'Css',
+    tag3: 'Javascript',
+    sourceLink: '',
+    demoLink: '',
+  },
```

Comment:
- [ ] The modal popup is fixed on page reload and cannot be gotten rid of by clicking the `X` button, making it imposible to get to the `Contac Me` section of the page for review. Kindly fix this and someone will review your work.
![image](https://user-images.githubusercontent.com/81932478/180606424-cd484fa3-588d-450d-8960-6db621aad4bf.png)
This is what your page looks like on load.